app/src/main/java/org/oppia/android/app/player/state/answerhandling/InteractionAnswerHandler.kt

```diff
@@ -26,6 +27,11 @@ interface InteractionAnswerHandler {
   fun getPendingAnswer(): UserAnswer? {
     return null
   }
+
+  /** Return the last answer entered by user. */
+  fun getRawUserAnswer(): RawUserAnswer {
+    return RawUserAnswer.getDefaultInstance()
```

Comment:
Are you sure its correct, ie. ` /** Return the last answer entered by user. */` because here it is returning `default` always.

Reply:
`/** Return the last answer entered by user. */` is correct because here this function is override in every StateItemViewModel to get last entered answer.

Reply:
Okay.